files/build_templates/sonic_debian_extension.j2

```diff
@@ -278,10 +269,15 @@ sudo rm -f $FILESYSTEM_ROOT/usr/sbin/policy-rc.d
 
 # Copy fstrim service and timer file, enable fstrim timer
 sudo cp $IMAGE_CONFIGS/fstrim/* $FILESYSTEM_ROOT/etc/systemd/system/
-sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable fstrim.timer
 
 ## copy platform rc.local
-sudo cp $IMAGE_CONFIGS/platform/rc.local $FILESYSTEM_ROOT/etc/
+j2 $IMAGE_CONFIGS/platform/rc.local.j2 | sudo tee $FILESYSTEM_ROOT/etc/rc.local
+sudo chmod +x $FILESYSTEM_ROOT/etc/rc.local
```

Comment:
does not seem neccessary for rc.local.

Reply:
Forgot to revert this change, will do so.